DISPLAY "(null)" invalid; disabling X11 forwarding
INFO:tensorflow:PS hosts are: ['raiders1:2200']
INFO:tensorflow:Worker hosts are: ['raiders1:2201', 'raiders1:2202', 'raiders1:2203', 'raiders1:2204', 'raiders2:2205', 'raiders2:2206', 'raiders2:2207', 'raiders2:2208', 'raiders3:2209', 'raiders3:2210', 'raiders3:2211', 'raiders3:2212', 'raiders8:2213', 'raiders8:2214', 'raiders8:2215', 'raiders8:2216']
I tensorflow/core/distributed_runtime/rpc/grpc_channel.cc:206] Initialize HostPortsGrpcChannelCache for job ps -> {raiders1:2200}
I tensorflow/core/distributed_runtime/rpc/grpc_channel.cc:206] Initialize HostPortsGrpcChannelCache for job worker -> {raiders1:2201, raiders1:2202, raiders1:2203, raiders1:2204, raiders2:2205, raiders2:2206, raiders2:2207, raiders2:2208, raiders3:2209, raiders3:2210, raiders3:2211, raiders3:2212, localhost:2213, raiders8:2214, raiders8:2215, raiders8:2216}
I tensorflow/core/distributed_runtime/rpc/grpc_server_lib.cc:202] Started server with target: grpc://localhost:2213
INFO:tensorflow:Learning rate: 0.002500, momentum: 0.900000
INFO:tensorflow:Sync mode!!!!!!
INFO:tensorflow:Compute groups : 1
INFO:tensorflow:SyncReplicas enabled: replicas_to_aggregate=16; total_num_replicas=16
INFO:tensorflow:Num nodes per compute groups: 16
INFO:tensorflow:Tokens per step: 16
INFO:tensorflow:2016-07-26 08:39:17.544916 Supervisor
E0726 08:39:29.359742465   68127 tcp_client_posix.c:191]     failed to connect to 'ipv4:172.24.75.45:2209': timeout occurred
E0726 08:39:29.359845288   68127 tcp_client_posix.c:191]     failed to connect to 'ipv4:172.24.75.45:2210': timeout occurred
E0726 08:39:29.359866074   68127 tcp_client_posix.c:191]     failed to connect to 'ipv4:172.24.75.45:2211': timeout occurred
E0726 08:39:29.359882346   68127 tcp_client_posix.c:191]     failed to connect to 'ipv4:172.24.75.45:2212': timeout occurred
E0726 08:39:30.098179034   67797 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.45:2209': socket error: 113
E0726 08:39:30.098267310   67797 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.45:2210': socket error: 113
E0726 08:39:30.098292653   67797 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.45:2211': socket error: 113
E0726 08:39:30.098312219   67797 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.45:2212': socket error: 113
E0726 08:39:33.098101411   67797 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.45:2211': socket error: 113
E0726 08:39:33.098157680   67797 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.45:2212': socket error: 113
E0726 08:39:33.098180584   67797 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.45:2209': socket error: 113
E0726 08:39:33.098231388   67797 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.45:2210': socket error: 113
E0726 08:39:39.098062737   67797 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.45:2212': socket error: 113
E0726 08:39:39.098131536   67797 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.45:2211': socket error: 113
E0726 08:39:39.098149555   67797 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.45:2210': socket error: 113
E0726 08:39:39.098194986   67797 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.45:2209': socket error: 113
E0726 08:39:45.097988535   67797 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.45:2212': socket error: 113
E0726 08:39:45.098036469   67797 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.45:2211': socket error: 113
E0726 08:39:48.098012462   67797 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.45:2210': socket error: 113
E0726 08:39:51.097985059   67797 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.45:2209': socket error: 113
E0726 08:39:54.097989096   67797 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.45:2212': socket error: 113
E0726 08:39:57.097979695   67797 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.45:2211': socket error: 113
E0726 08:40:00.098008167   67797 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.45:2210': socket error: 113
E0726 08:40:03.097927662   67797 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.45:2209': socket error: 113
E0726 08:40:09.605990104   67797 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.45:2212': socket error: 113
E0726 08:40:18.605991676   67797 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.45:2211': socket error: 113
E0726 08:40:24.605973663   67797 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.45:2209': socket error: 113
E0726 08:40:24.606019906   67797 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.45:2210': socket error: 113
E0726 08:40:33.521965339   67797 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.45:2212': socket error: 113
E0726 08:41:00.294019133   67797 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.45:2209': socket error: 113
E0726 08:41:00.294083061   67797 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.45:2211': socket error: 113
run-expr-32node.sh: line 23: 67530 Terminated              CUDA_VISIBLE_DEVICES='' bazel-bin/inception/imagenet_distributed_train --batch_size=16 --data_dir=/lfs/local/0/daniter/16node-data-$2 --job_name='worker' --task_id=$2 --ps_hosts='raiders1:2200' --worker_hosts='raiders1:2201,raiders1:2202,raiders1:2203,raiders1:2204,raiders2:2205,raiders2:2206,raiders2:2207,raiders2:2208,raiders3:2209,raiders3:2210,raiders3:2211,raiders3:2212,raiders8:2213,raiders8:2214,raiders8:2215,raiders8:2216' --initial_learning_rate=$3 --momentum=$4 --sync=True --compute_groups=$5
